ui/src/app/pages/access/identity-help-text.tsx

```diff
@@ -38,7 +38,7 @@ const ContactUs = (props: { profile: Profile }) => {
 
 export const IdentityHelpText = (props: {
   profile: Profile;
-  afterInitialClick: boolean;
+  afterInitialClick?: boolean;
```

Comment:
Many of the typing updates document pre-existing usage. One of the commits addresses this specifically.